roles/fs-drift/tasks/main.yml

```diff
@@ -0,0 +1,85 @@
+---
+- set_fact:
+    fs_drift_path: /mnt/pvc
+
+- name: Create PVCs
+  k8s:
+    definition:
+      kind: PersistentVolumeClaim
+      apiVersion: v1
+      metadata:
+        name: "fs-drift-claim-{{ item }}"
+        namespace: "{{ operator_namespace }}"
+        annotations:
```

Comment:
we can put this file in a template and add a check condition to use a specific storage class only when fs_drift.storageclass is defined so that if user doesn't provide a name it'll auto pick the one with annonation default